src/snyk/snykOss/providers/ossDetailPanelProvider.ts

```diff
@@ -78,17 +79,32 @@ export class OssDetailPanelProvider
       }, {});
 
       const displayMode = 'dark';
+
+      let html = issue.additionalData.details;
+
+      // Add the style
+      const ideStylePath = vscode.Uri.joinPath(
+        vscode.Uri.file(this.context.extensionPath),
+        'media',
+        'views',
+        'oss',
+        'suggestion',
+        'suggestion.css',
+      );
+      const ideStyle = readFileSync(ideStylePath.fsPath, 'utf8');
+      const nonce = getNonce();
+
+      // TODO: remove after the stable CLI release at the end of cycle 5
```

Comment:
Reminder to create a ticket to remove this code when we fully enable this HTML panel after the end of cycle 5 and the stable release of the CLI